app/controller_services/shopping_list_items_controller/destroy_service.rb

```diff
@@ -22,7 +22,6 @@ def perform
 
       ActiveRecord::Base.transaction do
         shopping_list_item.destroy!
-        shopping_list.touch
```

Comment:
Should no longer be needed but we'll double check at the point where we make these tests pass again.